tree/ntuple/v7/inc/ROOT/RFieldToken.hxx

```diff
@@ -0,0 +1,56 @@
+/// \file ROOT/RFieldToken.hxx
+/// \ingroup NTuple ROOT7
+/// \author Jonas Hahnfeld <jonas.hahnfeld@cern.ch>
+/// \date 2015-03-19
+/// \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
+/// is welcome!
+
+/*************************************************************************
+ * Copyright (C) 1995-2025, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT7_RFieldToken
+#define ROOT7_RFieldToken
+
+#include <cstdint>
+
+namespace ROOT {
+namespace Experimental {
+
+class REntry;
+class RNTupleModel;
+
+namespace Detail {
+class RRawPtrWriteEntry;
+} // namespace Detail
+
+// clang-format off
+/**
+\class ROOT::Experimental::RFieldToken
```

Comment:
Shall I move the class out of `Experimental` directly? @silverweed

Reply:
I think so. But maybe we can double check with @jblomer to see if he has any objection to this.

Reply:
I rebased the PR, resolved the conflicts, and moved to `ROOT::RFieldToken`.